src/Animations/Animations/Animations/BounceAnimation.cs

```diff
@@ -0,0 +1,82 @@
+﻿namespace FormsCommunityToolkit.Animations
+{
+    using System;
+    using System.Threading.Tasks;
+    using Xamarin.Forms;
+
+    public class BounceInAnimation : AnimationBase
+    {
+        protected override Task BeginAnimation()
+        {
+            if (Target == null)
+            {
+                throw new NullReferenceException("Null Target property.");
+            }
+
+            return Task.Run(() =>
+            {
+                Device.BeginInvokeOnMainThread(() =>
+                {
+                    Target.Animate("BounceIn", BounceIn(), 16, Convert.ToUInt32(Duration));
+                });
+            });
+        }
+
+        internal Animation BounceIn()
```

Comment:
Does this have to be internal or can it be private? (haven't opened the source to check references). Similar remark on the other animations.